install.sh

```diff
@@ -37,6 +37,10 @@ else
     exit
 fi
 
+echo "Building mariadb-udf image"
+
+make -C ./mariadb-udf-docker/
```

Comment:
You need to check if `make` and `g++` is available in the system. This type of check is done Line 4. Edit that.